src/library/styles/pxToEm.js

```diff
@@ -4,8 +4,20 @@
  */
 import fontSize_base from '../themes/fontSizeBase';
 
+const justNumbersOrEndsWithPx = (s: string) => s.match(/^[\d.]+(px)?$/);
+
+const convert = (input: number): string =>
+  `${input / parseInt(fontSize_base)}em`;
+
+const coerce = (value) => (typeof value === 'string' ? value : typeof value);
```

Comment:
`coerce` could prolly be named better - `valueOrType` perhaps?  Also, it probably does not need to be a separate function if it is only called once.  Consider just creating a const inside the error function.